ehrapy/preprocessing/_scanpy_pp_api.py

```diff
@@ -225,6 +232,23 @@ def neighbors(
         method: Use 'umap' [McInnes18]_ or 'gauss' (Gauss kernel following [Coifman05]_ with adaptive width [Haghverdi16]_) for computing connectivities.
                 Use 'rapids' for the RAPIDS implementation of UMAP (experimental, GPU only).
         metric: A known metric’s name or a callable that returns a distance.
+        transformer
+            Approximate kNN search implementation following the API of
+            :class:`~sklearn.neighbors.KNeighborsTransformer`.
+            See :doc:`/how-to/knn-transformers` for more details.
+            Also accepts the following known options:
+
+            `None` (the default)
+                Behavior depends on data size.
+                For small data, we will calculate exact kNN, otherwise we use
+                :class:`~pynndescent.pynndescent_.PyNNDescentTransformer`
+            `'pynndescent'`
+                :class:`~pynndescent.pynndescent_.PyNNDescentTransformer`
+            `'rapids'`
+                A transformer based on :class:`cuml.neighbors.NearestNeighbors`.
```

Comment:
Here can either add "deprecated since scanpy version 1.10.0" such that its clear where 1.10.0 comes from, or hide this from the beginning as its deprecated already?
Then, would also need to remove from `_KnownTransformer` if we write this ourselves see question on what to do about private variables below